tests/main/base-policy/test-snapd-sh-other18/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: test-snapd-sh-other18
+version: 1.0
+architectures: ["all"]
+base: test-snapd-core18
```

Comment:
Did you intend to add this snap but forgot?

Reply:
I thought it would auto-resolve and it is add since it passed here. Maybe it is because I ran spread with -reuse, then moved the dir, then reran the spread test (ie, so it downloaded already)? I'll explicitly install it.

Reply:
I'm installing manually now.

Reply:
the snap exists in the store, not sure what is going on, it should be installed automatically but something isn't working

Reply:
@pedronis - I saw an i386 failure a moment ago-- the test-snapd-core18 snap is only available on amd64. I bet that is what @zyga looked at and why it didn't install automatically (the logs are gone now).

Reply:
mmh, you'll have to talk with @sergiocazzolato and/or @mvo5 how to get a i386 version uploaded (and maybe arm* ones, not sure)